docs/05-technical-reference/00-architecture/apix-01-architecture-apigateway.md

```diff
@@ -6,9 +6,9 @@ This diagram illustrates the workflow that leads to exposing a service in Kyma:
 
 ![service-exposure-flow](./assets/apix-api-gateway-flow.svg)
 
-- **API Gateway Controller** is a component responsible for exposing services. API Gateway Controller is an application deployed in the `kyma-system` Namespace, implemented according to the [Kubernetes Controller](https://book.kubebuilder.io/) principles. API Gateway Controller listens for newly created custom resources (CR) that follow the set `apirule.gateway.kyma-project.io` custom resource definition (CRD), which describes the details of exposing services in Kyma.
+- **API Gateway Controller** is a component responsible for exposing services. API Gateway Controller is an application deployed in the `kyma-system` Namespace, implemented according to the [Kubernetes Controller](https://book.kubebuilder.io/) principles. API Gateway Controller listens for newly created custom resources (CR) that follow the set `.gateway.kyma-project.io` custom resource definition (CRD), which describes the details of exposing services in Kyma.
```

Comment:
```
(...) that follow the set `.gateway.kyma-project.io`(...)
```

Just double-checking: is this change (removal of the `apirule` prefix) intentional?

Reply:
Nope, thanks for checking :)